app/controllers/AdminController.php

```diff
@@ -1,137 +1,165 @@
 <?php
-//require_once 'models/PostModel.php';
-//require_once 'models/CommentModel.php';
-//require_once 'models/UserModel.php';
 namespace app\controllers;
-use app\models\CommentModel;
-use app\models\PostModel;
-use app\models\UserModel;
+use app\models\Comment;
+use app\models\Post;
+use app\models\User;
 use app\utils\View;
+use Exception;
 
 class AdminController {
 
-    private PostModel $postModel;
-    private CommentModel $commentModel;
-    private UserModel $userModel;
+    private Post $post;
+    private Comment $comment;
+    private User $user;
 
     public function __construct() {
-        $this->postModel = new PostModel();
-        $this->commentModel = new CommentModel();
-        $this->userModel = new UserModel();
+        $this->post = new Post();
```

Comment:
One thing you could consider here is property promotion:  https://stitcher.io/blog/constructor-promotion-in-php-8